src/App.js

```diff
@@ -1,17 +1,18 @@
 import React, { useState, useEffect } from 'react';
-import { StyleSheet, Text, TouchableOpacity, View } from 'react-native';
+import { StyleSheet, Text, SafeAreaView, View, useWindowDimensions } from 'react-native';
 import AnimateSvg from './AnimateSvg';
 import AnimateQuotes from './AnimateQuotes';
 import { useIsShake } from './useIsShake';
 
 export default function App() {
   const [hasInitialTransitionRun, setHasInitialTransitionRun] = useState(false);
   const { isShakeTriggered, setIsShakeReady } = useIsShake();
+  const { height } = useWindowDimensions();
```

Comment:
In mobile devices that can modify app window size in relationship to screen size, `height` will update dynamically.